cypress/tests/t.spec.ts

```diff
@@ -7,5 +7,6 @@ describe('Checks if auth is setup', () => {
   });
   it('Visits the app root url', () => {
     cy.visit('/');
+    cy.log('Foobar'); // [Debug]
```

Comment:
![image](https://user-images.githubusercontent.com/33557095/177148173-89d16047-9521-462e-bf85-4c59b61ad97c.png)

Reply:
Removing this line now.

Reply:
(this was just to demonstrate that the image for testing was being created and used from the PR)